### ISSUE TITLE
Why is there no decimal editor?

### ISSUE THREAD

**Issue by Rodbjartson** [olifolkerd/tabulator#779]
Hey there!
Why is there no possibility to use an option to edit decimals.
Thats is float datatypes such as 1,88 etc.
Say I want to make an ajax request, and store values such as 200,54.
When I use editor:number, this replaces the float value with 0...
Obviously, when you ask someone to input a float, you will restrict them to values such as "3", "777,33" etc. and not let them type in "hello".

**Comment by olifolkerd** [comment]
Hey @Rodbjartson 

Thanks for getting in touch, i assume you mean to ask "could someone add a decimal editor".

The development of Tabulator is driven by the community with requested features being added as they gain popularity in the issues list.

If you would like an example of how to write a custom editor for this (you can add any editor functionality you like to Tabulator) then you just has to ask. and you can raise a feature or pull request yourself if you would like to add the feature to Tabulator.

Please approach these issue lists with a more open minded attitude and writing style, while something may seem like a basic requirement from your point of view, it is actually the first time that a request for this feature has been made.

Cheers

Oli

**Comment by olifolkerd** [comment]
Hey @Rodbjartson 

You will be happy to hear i have just added max, min and step options to the editorParamsfor the number editor on the 3.4 branch:

```js
{title:"Value", field:"value", editor:"number", editorParams:{min:0, max:100, step:0.01}},
```

This will be included in the 4.2 release in a couple of weeks time.

Cheers

Oli :)

**Comment by olifolkerd** [comment]
Tabulator version 3.4 has now been officially release!

You can find details of the new release in the [News Section](http://tabulator.info/news)

Full documentation of all the new features can be found [Here](http://tabulator.info/docs/3.4)

Cheers

Oli :)
